### ISSUE TITLE
Cannot add usev4/webv4 as local variables

### ISSUE THREAD

**Issue by harvii** [ddclient/ddclient#708]
Using this below config results in the warning `WARNING:  Could not determine an IP for mydomain.ddns.net` and no IP is found for `mydomain.ddns.net`:
```
daemon=300
syslog=yes
ssl=yes
pid=/run/ddclient/ddclient.pid
cache=/run/ddclient/ddclient.cache
web=googledomains
use=no

usev4=webv4, \
webv4=https://ipinfo.io/ip, \
protocol=noip, \
login=myusername, \
password='mypassword' \
mydomain.ddns.net
```
Here is the complete output of `ddclient -file /run/ddclient/ddclient.conf -daemon=0 -debug -verbose -noquiet`: [output1](https://gist.github.com/harvii/0e23287cc6110a459854f1d578bcec20)

However, using this config, an IP is found correctly but `usev4` and `webv4` are added as Global variables:
```
daemon=300
syslog=yes
ssl=yes
pid=/run/ddclient/ddclient.pid
cache=/run/ddclient/ddclient.cache
web=googledomains
use=no

usev4=webv4, \
webv4=https://ipinfo.io/ip,
protocol=noip, \
login=myusername, \
password='mypassword' \
mydomain.ddns.net
```
Here is the complete output of `ddclient -file /run/ddclient/ddclient.conf -daemon=0 -debug -verbose -noquiet`: [output2](https://gist.github.com/harvii/beaf2c1834094d13e4e7a7dd74264da9)

Is there a special format or syntax for adding local variables?

**Comment by rhansen** [comment]
What version of ddclient are you using?  Have you tried a recent version from the `master` branch?

**Comment by harvii** [comment]
Sorry, I forgot to mention it's version 3.11.2, specifically the version included with VyOS [1.5-rolling-202407100021](https://github.com/vyos/vyos-rolling-nightly-builds/releases/tag/1.5-rolling-202407100021).

**Comment by rhansen** [comment]
Duplicate of #585
